.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
Yes and BTW once this env is setted, this can't be viewed anymore, even if by Travis admin! :-)

Do you want to try on this bundle for test? I'll update the Travis config in consequence.